posthog/queries/person_query.py

```diff
@@ -287,6 +288,11 @@ def _get_search_clauses(self, prepend: str = "") -> Tuple[str, str, Dict]:
                 WHERE distinct_id = %({distinct_id_param})s
             )
             """
+            try:
+                UUID(self._filter.search)  # will this throw?
+                distinct_id_clause = f"(id = %({distinct_id_param})s OR {distinct_id_clause})"
```

Comment:
For clarity it would be better if `distinct_id_clause = ...` were in the `else:` clause of the `try:except:`. It would make the "will this throw?" comment redundant.

Also a nit: this is no longer the `distinct_id` clause, I think `id_conditions_sql` would better reflect that this covers both types of ID and would better match the convention from the rest of the function